qiskit/visualization/pulse/matplotlib.py

```diff
@@ -146,13 +156,17 @@ def to_table(self, name):
         for key, val in snapshots.items():
             data_str = 'snapshot: %s' % val
             time_event.append((key, name, data_str))
+        for key, val in frequencychanges.items():
+            data_str = 'frequency: %.4f' % val
```

Comment:
while automerge is running, I think it is better to use `.4e` because now frequency is returned in the unit of Hz and typical frequency range depends on the system.

Reply:
Resolved in 50440fc